### PR TITLE
Register windowsed.is-a.dev

### PR DESCRIPTION
**Pull request by windowsedd**
Register windowsed.is-a.dev with A record pointing to 76.76.21.21.

### CODE REVIEW

File: domains/windowsed.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "windowsed1225",
+           "email": "",
+           "discord": "906415095473655810",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.b4wkSB9hzfyZzAOO50m5Lmffz5ZpdPkeT6RBILTZckY94cbRa7BxRCVOAevnxu4zqonBcx017nf0Kgu1qP58hQV8rbPZM7bEeJwIuPh-gY_l1y_IkzE5Slq1CWvmFCBZvheUKYHFvPRcGGIqtxyKKaT6jcrJXv7xo_F2fldqbJ2NiJ88_u4WTXcfQ9CrPrPevocKfdesIrDwXnV2n2I0SpDV_QqycbmCcleRy3QzOpZMTHQyzAV_WA0B8OEkMftVSbiTr-S4WPRMpoZTRX7Er-l-ccC5HIs1c7P-twpXd4bphTsUBmWJL55dD0xq7vDQ2ouWq-o4ooqqZ3wmme-KvA.R7TKz12p_AeXBaX-7nEu9g.tjS2Ho_9HJkfxTIoz2MFeX34ptUfBAIh6vPSR_lMepN4Nz03-n_JxdpZW-5iEmDqBUUU82h7WteWeRKhyBkvi1syxFOnMRTBT4EveG-Ij9-skVlWjzeAQw-6WWRaGtDPfBSyZsTKyIb9iI2tVP7Kvw.jJ5K9-F9Rh-7DMlxCJ8yQA"
+        },
+    
+        "record": {
+            "A": ["76.76.21.21"]
```
Comment (CuteDog5695) [inline domains/windowsed.json:10]: You need to just add Vercel TXT record, follow https://www.is-a.dev/docs/vercel/ for more information

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
Putting this label since i dunno what else to put, might make another label later

File: domains/windowsed.json (hunk 1 above)
Comment (windowsed1225) [reply domains/windowsed.json:10]: image

Comment (windowsed1225) [reply domains/windowsed.json:10]: huh lol?

**Review by iostpa — CHANGES_REQUESTED** [review]
Just realised you got `_vercel.windowsed.json` merged without this subdomain. I guess i just need a preview of your site then

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**